react/src/components/atoms/forms/InputTextTypeAhead/InputTextTypeAhead.stories.js

```diff
@@ -24,7 +24,8 @@ storiesOf('atoms/forms', module).addDecorator(withKnobs)
           inputOptions.options.orgSelector.map((option) => option.text),
           ''
         ),
-        onChange: action('InputTextTypeAhead onChange')
+        onChange: action('InputTextTypeAhead onChange'),
+        disabled: boolean('InputTextTypeAhead.disabled', false)
```

Comment:
It's a bit nit-picky and certainly not a merge blocker, but it might make more sense to use `enabled` as your boolean than `disabled`, when true === enabled and false === disabled, it's easier to not have to think about later.

Reply:
agreed